terraform/custom_domains/environmental_domains/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "zone" {}
+variable "front_door_name" {}
+variable "resource_group_name" {}
+variable "domains" {}
+variable "environment_short" {}
+variable "app_name" {}
+variable "app_resource_group_name" {}
+variable "app_sub_id" {}
+variable "redirect_rules" {
+  default = {}
+}
```

Comment:
Could we add a `type` and `description` to these variables?

Reply:
Updated with type and description.